src/App.vue

```diff
@@ -25,12 +29,45 @@
   var row3: number = getRandomNumbersInRange(1,0,24)
   var row4: number = getRandomNumbersInRange(1,0,24)
   var row5: number = getRandomNumbersInRange(1,0,24)
+
+  const [isModalVisible, toggleModal] = useToggle()
   
 </script>
 
+<script lang="ts">
+
+</script>
+
 <template>
-  <Header msg="scoopy" />
-  <section class="module-row">
+
+  <header class="main-header">
```

Comment:
This should probably be put into Header.vue instead of App.vue. Will also help with merge conflicts probably.